### ISSUE TITLE
Jacobian

### ISSUE THREAD

**Issue by hsk9767** [zhujiapeng/resefa#1]
Hi, 

When you get J^(T)J, do you make it a 'Low-rank matrix' as your previous paper, 'Low-Rank Subspaces in GANs'?

(J means jacobian and J^(T) means transpose of J)

Thank you.

**Comment by zhujiapeng** [comment]
No, the J^TJ on the background needs to be full rank since it is the denominator.
